components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Model/MonitoredItemModelTests.cs

```diff
@@ -61,7 +61,8 @@ public class MonitoredItemModelTests {
                         AttributeId = NodeAttribute.DataType,
                         BrowsePath = new string[] { "EventBrowsePath "},
                         IndexRange = "SelectClauseIndexRange",
-                        TypeDefinitionId = "SelectClauseTypeDefinitionId"
+                        TypeDefinitionId = "SelectClauseTypeDefinitionId",
```

Comment:
Any additional asserts needed for the tests (e.g., for DisplayName)?

Reply:
No, it's done in the Equals / and IsSameAs) methods